apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/matcher/SpringAntPathMatcher.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.skywalking.apm.plugin.trace.ignore.matcher;
+
+import org.springframework.util.AntPathMatcher;
+import org.springframework.util.PathMatcher;
+
+public class SpringAntPathMatcher implements TracePathMatcher {
+
+    private PathMatcher pathMatcher = new AntPathMatcher();
```

Comment:
If you are using the Spring matcher, then the plugin has to include the Spring jar into it. And you put `<artifactId>spring-core</artifactId>` provided in your pom. So who can make the spring core jar in the agent class loader?

Reply:
The further question is, is using spring core matcher a good idea? Include such a large library to solve this feature.

Reply:
I've had this thought too, I will not depend on the spring framework

Reply:
I will add a simple version of AntPathMatcher